README.md

```diff
@@ -147,3 +190,28 @@ The transformed paths of the two configuration files in the NGINX for Azure depl
 |--------------------------------------|-----------------------------------|---------------------------------------------|
 | /etc/nginx/nginx.conf                | /config/nginx.conf                | /etc/nginx/nginx.conf                       |
 | /etc/nginx/sites-enabled/mysite.conf | /config/sites-enabled/mysite.conf | /etc/nginx/sites-enabled/mysite.conf        |
+
```

Comment:
Feels like this section should be included before the portion showing how to do a combined update of config and certificates. At least my understanding is that users usually want to find docs. One part at a time to understand and then a combined solution that is best practice.